### PR TITLE
chore: workaround for flaky e2e tests

### PR DESCRIPTION
**Pull request by tomaszpowroznik**

### CODE REVIEW

File: e2e/helpers/hooks_helper.js [hunk 1]
```diff
@@ -13,7 +13,7 @@ module.exports = class HooksHelpers extends Helper {
     recorder.retry({
       retries: 10,
       minTimeout: 1000,
-      when: err => err.message.indexOf('Execution context was destroyed') > -1,
+      when: err => err.message.indexOf('Execution context was destroyed') > -1 || err.message.indexOf('Node is') > -1,
```
Comment (lewisbirks) [inline e2e/helpers/hooks_helper.js:16]: So is this going to retry when the message contains these strings?

**Review by lewisbirks — DISMISSED** [review]
Looks fine based on the problem you have described

File: e2e/helpers/hooks_helper.js (hunk 1 above)
Comment (tomaszpowroznik) [reply e2e/helpers/hooks_helper.js:16]: yes, there were 2 messages
Node is either not visible or not an HTMLElement
and one I dont remember something like Node id detached
will find exact error and fix this one

Comment (tomaszpowroznik) [reply e2e/helpers/hooks_helper.js]: done

**Review by lewisbirks — APPROVED** [review]

**Review by samuelzerozero — APPROVED** [review]

**Pull request merged**